guides/common/sass/components/_asciidoc.scss

```diff
@@ -0,0 +1,1489 @@
+  margin-bottom: $panel-margin-bottom - em-calc(10);
+}
+
+.sect1 {
+  // reduce padding below small breakpoint to match header font size change
+  padding-bottom: $panel-margin-bottom - em-calc(10);
+}
+
+@media #{$small} {
+  #content {
+    margin-bottom: $panel-margin-bottom;
+  }
+
+  .sect1 {
+    padding-bottom: $panel-margin-bottom;
+  }
+}
+
+.sect1:last-child {
+  padding-bottom: 0;
+}
+
+.sect1 + .sect1 {
+  border-top: $sect-divider-size $sect-divider-style $sect-divider-color;
+}
+
+// QUESTION why #content here?
+#content #{headers()} {
+  > a.anchor {
+    position: absolute;
+    z-index: 1001;
+    // 1ex is the width of "x"
+    width: 1.5ex;
+    margin-left: -1.5ex;
+    display: block;
+    text-decoration: none !important;
+    visibility: hidden;
+    text-align: center;
+    font-weight: normal;
+
+    &:before {
+      content: "\00A7";
+      font-size: 0.85em;
+      display: block;
+      // padding-top highly dependent on font
+      padding-top: 0.1em;
+    }
+  }
+
+  &:hover > a.anchor,
+  > a.anchor:hover {
+    visibility: visible;
+  }
+
+  > a.link {
+    color: $header-font-color;
+    text-decoration: none;
+    &:hover {
+      color: darken($header-font-color, 5%);
+    }
+  }
+}
+
+// AsciiDoc block styles
+
+// these blocks don't receive a specialized margin
+details,
+.audioblock,
+.imageblock,
+.literalblock,
+.listingblock,
+.stemblock,
+.videoblock {
+  margin-bottom: $panel-margin-bottom;
+}
+
+details > summary:first-of-type {
+  cursor: pointer;
+  display: list-item;
+  outline: none;
+  margin-bottom: 0.75em;
+}
+
+.admonitionblock td.content,
+.audioblock,
+.exampleblock,
+.imageblock,
+.listingblock,
+.literalblock,
+.stemblock,
+.openblock,
+.paragraph,
+.quoteblock,
+table.tableblock,
+.verseblock,
+.videoblock,
+.dlist,
+.olist,
+.ulist,
+.qlist,
+.hdlist {
+  > .title {
+    @extend %subheader;
+    text-rendering: $subheader-text-rendering;
+    text-align: left;
+    @if $subheader-font-family {
+      font-family: $subheader-font-family;
+    }
+    @if $subheader-font-size {
+      font-size: $subheader-font-size;
+    }
+    @if $subheader-font-style {
+      font-style: $subheader-font-style;
+    }
+  }
+}
+
+// trick to prevent caption from influencing width of an autowidth table
+table.tableblock.fit-content > caption.title {
+  white-space: nowrap;
+  width: 0;
+}
+
+// Firefox sizes caption to width of table parent, so setting width: 100% works
+// A better solution is to wrap the table in a figure/figcaption
+//@supports (-moz-appearance:none) {
+//  table.tableblock.fit-content > caption.title {
+//    white-space: normal;
+//  }
+//  table.tableblock:not(.stretch) > caption.title {
+//    width: 100%;
+//  }
+//}
+
+// matches %lead
+.paragraph.lead > p,
+#preamble > .sectionbody > [class="paragraph"]:first-of-type p {
+  font-size: $paragraph-font-size + em-calc(3.5);
+  line-height: 1.6;
+  color: $title-font-color;
+  // QUESTION remove or reduce letter-spacing?
```

Comment:
See above.

Reply:
same as above, not mine code, it's from ascii-doc team